packages/kbn-management/cards_navigation/src/consts.tsx

```diff
@@ -81,7 +81,7 @@ export const appDefinitions: Record<AppId, AppDefinition> = {
     category: appCategories.DATA,
     description: i18n.translate('management.landing.withCardNavigation.mlDescription', {
       defaultMessage:
-        'View, export, and import machine learning analytics and anomaly detection items.',
+        'Identify, analyze, and process areas of interest in your data using advanced analysis techniques.',
```

Comment:
@sabarasaba can you help me understand where this copy is being used? I didn't think we showed Machine Learning on the landing page 🤔